build/module-builder/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:12-alpine as builder
+WORKDIR /botpress
+RUN apk add --no-cache git
+RUN git clone https://github.com/botpress/botpress.git
+
+FROM node:12-alpine
+# still need git, because of the gulp initStudio task!?
```

Comment:
Please explain why this doesn't work without git installed if pro modules are not fetched!

Reply:
The Studio and Admin UI have dependencies on a fork of the `hard-source-webpack-plugin` that requires git to be installed. In their respective package.json files, you will find:

```
...
    "hard-source-webpack-plugin": "git+https://github.com/botpress/hard-source-webpack-plugin.git",
...
```
Installing `git+...` dependencies requires the git binary.